nvflare/app_common/pt/pt_file_model_locator.py

```diff
@@ -17,6 +17,7 @@
 from nvflare.apis.dxo import DXO, DataKind
 from nvflare.apis.event_type import EventType
 from nvflare.apis.fl_context import FLContext
+from nvflare.app_common.abstract.model import ModelLearnableKey, model_learnable_to_dxo
```

Comment:
Looks like the ModelLearnableKey is no longer needed. Please remove from the import.

Reply:
cleaned up.